backend/module/authn/authn.go

```diff
@@ -104,3 +106,21 @@ func (a *api) Callback(ctx context.Context, request *authnv1.CallbackRequest) (*
 		AccessToken: token.AccessToken,
 	}, nil
 }
+
+func (a *api) CreateToken(ctx context.Context, request *authnv1.CreateTokenRequest) (*authnv1.CreateTokenResponse, error) {
```

Comment:
i think calls to this endpoint need to fail if no storage is configured, otherwise you can create a permanently valid irrevocable token.

Reply:
That's a great point, let me make the change